DOC-1086 Merchant payment object general info (#351)
swan-io/docs: docs/topics/merchants/online/index.mdx

Your merchants can create links to share with their customers by email or text message.
They could also include a QR code on an invoice, scannable by the customer with their mobile device.

→ Learn how to [create payment links](./cards/guide-create-link.mdx) to accept card payments.

## Merchant payments {#payments}

Merchant card payments include **one or more transactions**.

> For example, one payment might include a capture transaction and a refund transaction, for a total of two transactions.

Reviewer: ```suggestion
For example, one payment might include both a capture and refund transaction.
```
